File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseRoutes.scala

package org.openapitools.client.model


case class ResponseRoutes (
    _results: List[ResponseRoutesResult]
)
object ResponseRoutes {
    def toStringBody(var_results: Object) =
        s"""
        | {
        | "results":$var_results
        | }
        """.stripMargin
}

File: dart-jaguar/lib/model/response_routes.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

import 'package:openapi/model/response_routes_result.dart';
part 'response_routes.jser.dart';

class ResponseRoutes {
  
  @Alias('results')
  final List<ResponseRoutesResult> results;
  

  ResponseRoutes(
    

{
    
     this.results = const [] 
    }
  );

  @override
  String toString() {
    return 'ResponseRoutes[results=$results, ]';
  }
}

@GenSerializer()
class ResponseRoutesSerializer extends Serializer<ResponseRoutes> with _$ResponseRoutesSerializer {

}
